scrapper.py

```diff
@@ -26,38 +41,183 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        link = article_bs.find('a')
+        return 'https://www.m24.ru' + link['href']
```

Comment:
do you only extract one URL from each seed URL?

Reply:
i fixed that